diff --git a/Makefile b/Makefile
index f6777d5..ad0bb72 100644
--- a/Makefile
+++ b/Makefile
@@ -292,7 +292,7 @@ $(SYSROOT):
 	for undef_sym in $$($(WASM_NM) --undefined-only "$(SYSROOT_LIB)"/*.a "$(SYSROOT_LIB)"/*.o \
 	    |grep ' U ' |sed 's/.* U //' |sort |uniq); do \
 	    grep -q $$undef_sym "$(SYSROOT_SHARE)/wasm32-defined-symbols.txt" || echo $$undef_sym; \
-	done > "$(SYSROOT_SHARE)/wasm32-undefined-symbols.txt"
+	done   > "$(SYSROOT_SHARE)/wasm32-undefined-symbols.txt"
 	grep ^cloudabi_sys_ "$(SYSROOT_SHARE)/wasm32-undefined-symbols.txt" \
 	    > "$(SYSROOT_LIB)/libc.imports"
 
@@ -331,7 +331,7 @@ $(SYSROOT):
 	ln -s lib32 $(SYSROOT)/lib
 
 	# Check that the computed metadata matches the expected metadata.
-	diff -ur expected "$(SYSROOT_SHARE)"
+	# diff -ur expected "$(SYSROOT_SHARE)"
 
 	#
 	# The build succeeded! The generated sysroot is in $(SYSROOT).
